frontend/public/components/start-guide.tsx

```diff
@@ -70,9 +68,7 @@ export const withStartGuide = (WrappedComponent, disable: boolean = true) =>
               <OpenShiftGettingStarted canCreateProject={flags[FLAGS.CAN_CREATE_PROJECT]} />
             </HintBlock>
           </div>
-          {// Whitelist certain resource kinds that should not be disabled when no projects are
-          // available. Disabling should also be optional
-          !disable || WHITELIST.includes(kind) ? (
+          {!disable || !kindObj.namespaced ? (
```

Comment:
Do we know if `kindObj` is always defined? We use `_.get` above. I'm wondering if there is an edge case where it might be undefined while API discovery is still happening since we don't block console load on discovery.